sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 12.24.0b1 (Unreleased)
 
 ### Features Added
-
+- Added new `metadata` keyword to the `put_block_from_url` API to allow metadata to be supplied in the same manner as in the `upload_blob` API and others.
```

Comment:
Open to suggestions- Technically, as stated in the GH issue, you could have always supplied metadata, but this change simply brings it in alignment with the other APIs that you can provide it in key-value pairs.

Reply:
Honestly, I would just say, "Added support for metadata to `upload_blob_from_url` via the new `metadata` keyword." (I don't think there is a need to say it's similar to anything. Also, the API is upload_blob_from_url.)